cilium-cli/cli/connectivity.go

```diff
@@ -286,6 +287,7 @@ func newCmdConnectivityPerf(hooks api.Hooks) *cobra.Command {
 func registerCommonFlags(flags *pflag.FlagSet) {
 	flags.BoolVarP(&params.Debug, "debug", "d", false, "Show debug messages")
 	flags.StringVar(&params.TestNamespace, "test-namespace", defaults.ConnectivityCheckNamespace, "Namespace to perform the connectivity in (always suffixed with a sequence number to be compliant with test-concurrency param, e.g.: cilium-test-1)")
+	flags.Var(option.NewNamedMapOptions("namespace-labels", &params.NamespaceLabels, nil), "namespace-labels", "Add labels to the connectivity test namespace")
```

Comment:
do we want to hide this new flag too? (what's the reason for hiding the others but not this one?)

Reply:
Not sure about the reasons why the others have been hidden. That said, I'd keep this one visible, because it is sensible for users to want to set a label in certain scenarios (e.g., to simplify retrieval/cleanup, or given that other projects rely on labels to enable certain features).